src/test/java/model/manager/RandomCarMoveManagerTest.java

```diff
@@ -0,0 +1,18 @@
+package model.manager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class RandomCarMoveManagerTest {
+
+    @ParameterizedTest
+    @CsvSource(value = {"0:false", "1:false", "2:false", "3:false", "4:true", "5:true", "7:true", "8:true",
+            "9:true"}, delimiter = ':')
+    void boundaryTest(String input, String expected) {
+        assertThat(new RandomCarMoveManager().isMove(Integer.parseInt(input))).isEqualTo(
+                Boolean.parseBoolean(expected));
```

Comment:
```suggestion
        assertThat(new RandomCarMoveManager().isMove(Integer.parseInt(input)))
        .isEqualTo(Boolean.parseBoolean(expected));
```
이런식으로 개행을 두면 조금 더 가독성이 좋아집니다!

Reply:
이부분!

```
    @ParameterizedTest
    @CsvSource(value = {"0:false", "1:false", "2:false", "3:false", "4:true", "5:true", "7:true", "8:true",
            "9:true"}, delimiter = ':')
    void boundaryTest(int input, boolean expected) {
        assertThat(new NumberCarMoveManager().isMove(input)).isEqualTo(expected);
    }
```
이렇게 만들어도 정상작동됩니다
`CsvSource`는 처음 입력될 때는 `String`인 것 같은데....
`parse`되는 형태로 작성하면 자동으로 parsing 되는 걸까요...?

Reply:
@eunkeeee 
말씀하신 부분에 대해 참고할만한 링크를 가져와 봤습니다! 문자열이 변환하고자 하는 타입과 형식이 일치하면 자동으로 형변환 해주는 것 같네요
https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests-argument-conversion-implicit

Reply:
좋은 레퍼런스를 가져오셨네요! 👍👍